README.md

```diff
@@ -0,0 +1,11 @@
+# Cincinnati
+
+Cincinnati is an update protocol designed to facility automatic updates. It describes a particular method for representing transitions between releases of a project and allowing a client to perform automatic updates between these releases.
+
+## Quick Start
+
+```sh
+# cargo run --package graph-builder -- --address 0.0.0.0 --registry https://quay.io --repository <namespace/reponame>
+# cargo run --package policy-engine -- --address 0.0.0.0 --upstream http://localhost:8080/v1/graph
+# curl --verbose --header 'Accept:application/json' http://localhost:8080/v1/graph
```

Comment:
This should probably be `localhost:8081` if it's the Policy Engine you are trying to hit.